run_pipeline.py

```diff
@@ -85,6 +110,27 @@ def test(data, model):
     return float(rmse), model
 
 
+def get_feature_info(full_data: HeteroData) -> tuple[FeatureInfo, FeatureInfo]:
+    customer_features = full_data.x_dict["customer"]
+    article_features = full_data.x_dict["article"]
+
+    customer_num_cat, _ = torch.max(customer_features, dim=0)
+    article_num_cat, _ = torch.max(article_features, dim=0)
```

Comment:
the number of categories is different for each feature, so we'll need to deal identify that per feature (vector dimension)

Reply:
thanks for explaining!